exchanges/gateio/gateio_websocket_request_futures.go

```diff
@@ -168,37 +168,11 @@ func (g *Gateio) WebsocketFuturesGetOrderStatus(ctx context.Context, contract cu
 	return &resp, g.SendWebsocketRequest(ctx, perpetualFetchOrderEPL, "futures.order_status", a, params, &resp, 1)
 }
 
-func getAssetFromFuturesPair(pair currency.Pair) (asset.Item, error) {
-	if pair.IsEmpty() {
-		return asset.Empty, currency.ErrCurrencyPairEmpty
-	}
-	switch pair.Quote.Item {
-	case currency.USDT.Item:
-		return asset.USDTMarginedFutures, nil
-	case currency.USD.Item:
-		return asset.CoinMarginedFutures, nil
-	default:
-		return asset.Empty, fmt.Errorf("%w futures pair: `%v`", asset.ErrNotSupported, pair)
-	}
-}
-
-// validateFuturesPairAsset enforces the asset.Item to be either USDT or Coin margined futures in relation to the pair
-// for correct routing.
-func validateFuturesPairAsset(pair currency.Pair, a asset.Item) error {
+// validateFuturesPairAsset enforces a futures contract pair quote currency matches an asset
+func validateFuturesPairAsset(a asset.Item, pair currency.Pair) error {
```

Comment:
```suggestion
func validateFuturesPairAsset(pair currency.Pair, a asset.Item,) error {
```

In sequence with the function name ("Pair Asset"), not "Asset Pair". This is also consistent with the exchange interface in which we use the same sequence for all funcs: https://github.com/thrasher-corp/gocryptotrader/blob/master/exchanges/interfaces.go#L32

Reply:
Yep. I'll be consistent about this going forwards.
I think about it the other way around in my head.

Fixed https://github.com/gbjk/gocryptotrader/commit/b6d6183b0d